---
title: C6103 | Microsoft Docs
ms.custom: 
ms.date: 11/04/2016
ms.reviewer: 
ms.suite: 
ms.technology: vs-ide-code-analysis
ms.tgt_pltfrm: 
ms.topic: article
f1_keywords: C6103
ms.assetid: 22d1ab35-31a3-4ba9-8ef4-7a64bce66621
caps.latest.revision: "4"
author: mikeblome
ms.author: mblome
manager: ghogen
ms.openlocfilehash: 96a9a47b96ba66acbd126fcbc9e2fea1d2cedc5c
ms.sourcegitcommit: fb751e41929f031d1a9247bc7c8727312539ad35
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 11/15/2017
---
# <a name="c6103"></a>C6103
Aviso C6103: retornando \<variável > de falha na chamada de função na linha \<local >.  
  
 Um caminho bem-sucedido por meio da função está retornando uma variável usada como um parâmetro `_Out_` para uma chamada de função interna malsucedida.  
  
 O problema talvez seja a função chamada e a função de chamada não anotadas por completo. A função de chamada pode exigir `_Always_`, `_Outptr_result_nullonfailure_` (`_COM_Outptr_` para código COM), ou uma anotação relacionada, e a função de chamada pode exigir uma anotação `_Success_`. Outra causa possível para esse aviso é que a anotação `_Success_` na função chamada está incorreta.  
  
## <a name="see-also"></a>Consulte também  
 [Usando anotações de SAL para reduzir defeitos de código do C/C++](../code-quality/using-sal-annotations-to-reduce-c-cpp-code-defects.md)